core/domain/exp_jobs_one_off_test.py

```diff
@@ -668,11 +978,390 @@ def test_migration_job_skips_deleted_explorations(self):
         with self.assertRaisesRegexp(Exception, 'Entity .* not found'):
             exp_services.get_exploration_by_id(self.NEW_EXP_ID)
 
+    def test_exploration_migration_job_produces_no_output(self):
+        """Test that Exploration Migration job produces no output."""
+        exp_domain.Exploration.create_default_exploration(
+            self.VALID_EXP_ID, title='title', category='category')
 
-class ExplorationStateIdMappingJobTest(test_utils.GenericTestBase):
```

Comment:
Not sure, but is github display kinda messed up? I don't think these lines should be deleted, But it looks like github shows it as an addition later on. So it's fine I think. Ditto for the above similar comment.

Reply:
Oh ok... No issues. I realised after leaving a couple of such comments that it indeed is a github display issue :). Thanks for confirming!